### DISCUSSION TITLE
Northbound fields invisible

### DISCUSSION THREAD

**Discussion by zierbeek** [tijsverkoyen/HomeAssistant-FusionSolar#174] [General]
Hi!  Thank you for this project. 

In the fusionsolar part, my installer has given my installeer permissions, however, i'm unable to see the northbound api tab. 
Would he need to do something different? 

Kind regards

**Comment by tijsverkoyen** [comment]
See https://github.com/tijsverkoyen/HomeAssistant-FusionSolar?tab=readme-ov-file#northbound-api--openapi

**Comment by jamos104** [comment]
You have to be the owner of the company to see this tab, not just an installer
